Cargo.toml

```diff
@@ -16,23 +16,23 @@ edition = '2018'
 anyhow = '1.0'
 thiserror = '1.0'
 lru_time_cache = '0.11'
-handlebars = '3.4'
+handlebars = '3.5'
 itertools = '0.9'
 json-patch = '0.2'
 log = '0.4'
 serde_derive = '1.0'
 serde_json = '1.0'
 shellexpand = '2.0'
-stderrlog = '0.4'
+stderrlog = '0.5'
 structopt = '0.3'
 tempfile = '3'
 url = '2'
 walkdir = '2'
 zip = '0.5'
 dogstatsd = '0.6'
-actix-web = '2'
+actix-web = '3.1'
```

Comment:
how big of a bump is this, should we be concerned about breakages?

Reply:
The major is due to some module renames